Sources/TasksExecutor.swift

```diff
@@ -7,7 +7,11 @@ public class TasksExecutor: TasksExecutorProtocol {
   ) {
 
     let task = session.dataTask(with: request) { (data, response, error) in
-      completion(data, error)
+        if(data?.count == 0){
+            completion(data, UrlError.jsonError("error 400"))
+        }else{
+            completion(data, error)
+        }
```

Comment:
If you want to detect a server error, you should have a look at https://developer.apple.com/forums/thread/114119 instead